flink-yarn/src/test/java/org/apache/flink/yarn/RegisterApplicationMasterResponseReflectorTest.java

```diff
@@ -81,28 +75,28 @@ public void testDoesntCallGetContainersFromPreviousAttemptsMethodIfAbsent() {
                 registerApplicationMasterResponseReflector.getContainersFromPreviousAttemptsUnsafe(
                         new Object());
 
-        assertThat(containersFromPreviousAttemptsUnsafe, empty());
+        assertThat(containersFromPreviousAttemptsUnsafe).isEmpty();
     }
 
     @Test
-    public void testGetContainersFromPreviousAttemptsMethodReflectiveHadoop22() {
+    void testGetContainersFromPreviousAttemptsMethodReflectiveHadoop22() {
         assumeTrue(
+                isHadoopVersionGreaterThanOrEquals(2, 2),
```

Comment:
could be removed since we use a newer version nowadays

Reply:
Done.